### DISCUSSION TITLE
Instant consumption

### DISCUSSION THREAD

**Discussion by Lukas1736** [sjdawson/TruckSimulatorPlugin#7] [Q&A] [answered]
you said that instant consumption is more like average consumption but i would love to have the right now consumption the consumption in this moment i also know that ther need to be some data about that because in nearly all trucks you can view it and i would love to use this in my dashboard

**Comment by sjdawson** [comment]
Hey @Lukas1736,

Sorry - pre-caffeine brain yesterday and I was getting mixed up 😩 . You're right, instant consumption isn't the same as average. However, the telemetry data the game outputs doesn't include instant consumption, only average. It might be  possible to calculate it from other telemetry values, but I don't think it'd ever match 1:1 with what the in game dashboards show. :(

**Reply by Lukas1736** [reply]
sad

**Reply by Lukas1736** [reply]
this is just wide thinking but could you not „just“ use the very accurate fuel value in liters and then how mutch this value changes in like 1 second and this value could you convert into L/100km with some maths sadly ther is no changed by amount function in the ncalc thingi

**Reply by sjdawson** [reply, marked as answer]
Yeah, that's where my thinking has been leading me, ultimately - instant consumption is just fuel used in a set (albeit very small) distance, I'll have a play - see if I can come up with anything that gets close to what's on the in-game dashboard. 👍🏻

**Reply by Lukas1736** [reply]
thx
